packages/yoga/src/Heading/native/Heading.jsx

```diff
@@ -0,0 +1,134 @@
+import { arrayOf, bool, func, shape, string } from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import { ArrowLeft } from '@gympass/yoga-icons';
+import Box from '../../Box';
+import Text from '../../Text';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import TitleContainer from './TitleContainer';
+
+const HeadingContainer = styled(Box)`
+  ${({
+    theme: {
+      yoga: {
+        components: { heading },
+      },
+    },
+  }) => `
+    padding: ${heading.paddingTop}px ${heading.paddingRight}px ${heading.paddingBottom}px ${heading.paddingLeft}px;
+`}
+`;
+
+const SubContainer = styled(Box)``;
```

Comment:
Hi @salomao-neto 
you can use the `Box` component directly on lines _53_ and _69_ since there no custom styles to them (: